algo-core/scheduler_utils.py

```diff
@@ -0,0 +1,148 @@
+import argparse
+import json
+import sys
+from pathlib import Path
+import jsonschema
+import pandas as pd
+
+
+# Input filenames:
+filename_onboarding = "onboarding_agents.txt"
+filename_mentors = "mentors.txt"
+filename_new = "new_agents.txt"
+input_folder = ""
```

Comment:
How about defaulting to the current directory?

```suggestion
input_folder = Path()
```

The `input_folder` is of the `Path` type later anyway.